proposals/dao/vip_7.ts

```diff
@@ -0,0 +1,97 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+
+/*
+
+Timelock Proposal #6
+
+Description: Pauses minting on FEI PSM, remove VOLT liquidity from FEI PSM, adds DAI PSM to PCVGuardian whitelist, unpause redemptions on the USDC PSM
+
+Steps:
+  1 - Pause Minting on the FEI PSM
+  2 - Remove all VOLT from FEI PSM
+  3 - Add DAI PSM to whitelisted addresses on PCV Guardian
+  4 - Unpause redemptions for USDC PSM
+
+*/
+
+const vipNumber = '7';
+
+const voltFloorPrice = '9000000000000000';
+const voltCeilingPrice = '10000000000000000';
+
+const daiReservesThreshold = ethers.utils.parseEther('10000000000');
+const mintLimitPerSecond = ethers.utils.parseEther('10000');
```

Comment:
add a comment here that this variable is unused currently because the PSM does not have the ability to mint.